File: examples/python/number/multiply.py
"""
Date Created:               2018-02-28
Date Modified:              2018-02-28
Version:                    1
Contract Hash:              a9448a2793dd32c3a453e8a83ebed7e3bb10e15e
Available on NEO TestNet:   False
Available on CoZ TestNet:   False
Available on MainNet:       False

Example:
    Test Invoke:            build /path/to/multiply.py test 0202 02 False False 2 5
    Expected Result:        10
    Operation Count:        49
    GAS Consumption:        0.044
"""


def Main(a, b):
    """
    :param a: First input number of concern
    :param b: Second input number of concern
    :type a: int
    :type b: int
    :return: Multiplication of the 2 input numbers
    :rtype: int
    """
    c = a * b  # It is a good practice to always allocate calculated value to a local variable instead of chaining to return or another function
    return c

File: examples/csharp/number/multiply.cs
using Neo.SmartContract.Framework;

// Date Created:                2018-03-06
// Date Modified:               2018-03-06
// Version:                     1
// Debug Contract Hash:         0x208d537c170d97927f30f1ad59039d0f50061de1 
// Release Contract Hash:       0x3df1e6c8226b1ea2c2c88e1a061a98d1071a1b87
// Available on NEO TestNet:    False
// Available on CoZ TestNet:    False
// Available on MainNet:        False

// Example  
// Input Params     :           0202
// Output Params    :           02
// Operation Count  :           Contract compiled in Debug: 60 / Contract compiled in Release: 34
// GAS Consumption  :           Contract compiled in Debug:     0.043 (NEO Debugger) / 0.054 (neo-gui-developer)  
// GAS Consumption  :           Contract compiled in Release:   0.024 (NEO Debugger) / 0.035 (neo-gui-developer)

namespace SumSmartContract
{
    public class Contract : SmartContract
    {
        public static int Main(int a, int b)
        {
            var c = a * b;
            return c;
        }
    }
}